docs/update-dependensies.md

```diff
@@ -0,0 +1,62 @@
+#  Updating Project Dependencies
+
+This document is about how to update the Pactus project repository to latest version.
+
+### Packages
+
+First of all you need to update golang dependencies to latest version using this commands:
+
+```sh
+go get -u ./...
+go mod tidy
+```
+Once all packages got updated, make sure you run `make build`, `make test` and `make build_gui` commands to make sure
+none of previous behaviors are broken. if any packages had breaking changes or some of them are deprecated, you need to
+update the code and use new methods or use another package.
+
+
+### Dev tools
+
+After packages, you need to update dev tools such as `golangci-lint`, `buf`, etc.
+
+You can go to root [make file](../Makefile) and find all dev tools on:
+
+```makefile
```

Comment:
This part in the Makefile is subject to change regularly, so it's better not to specify them in this document.